Casks/a/adobe-acrobat-reader.rb

```diff
@@ -1,11 +1,11 @@
 cask "adobe-acrobat-reader" do
-  version "24.002.20964"
-  sha256 "c45387a6c380db5d2a1e9cb45b13e3b9065b8a50c421a121964b5560b05270fb"
+  version "24.002.21005"
+  sha256 "55a53014b5e060658921f6d3a6d20fa64aae5b09fc9f6ea3d78e9c2378dab934"
 
   url "https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/#{version.no_dots}/AcroRdrDC_#{version.no_dots}_MUI.dmg"
   name "Adobe Acrobat Reader"
   desc "View, print, and comment on PDF documents"
-  homepage "https://acrobat.adobe.com/us/en/acrobat/pdf-reader.html"
+  homepage "https://get.adobe.com/reader/"
```

Comment:
```suggestion
  homepage "https://acrobat.adobe.com/us/en/acrobat/pdf-reader.html"
```

This change should be reverted since the `homepage` stanza now links to the application's download page rather than its website.

Reply:
Originally changed because the existing homepage URL was returning a 404, so `brew audit` failed. The original URL now works, so looks like the 404 was transient.

Reply:
I see. Let me check if the region has something to do with it.